packages/browser/types/bugsnag.d.ts

```diff
@@ -2,21 +2,21 @@ import * as BugsnagCore from "@bugsnag/core";
 
 // overwrite config interface, adding browser-specific options
 declare module "@bugsnag/core" {
-  interface IConfig {
```

Comment:
I guess these changes to the external type names constitute a breaking change.

Reply:
👍 yup this will be shipped in a major release with a whole bunch more breaking changes. Stay tuned!

Reply:
@bengourley @djskinner should we have an UPGRADING.md or whatever as part of this change that highlights that this is required? Should this be in the CHANGELOG as part of this PR?

Reply:
This PR is already merged but I'm happy to include changelog entries for subsequent PRs. Upgrading guide-wise I think that should be done once everything has landed on the `v71 integration branch.